dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -947,15 +947,15 @@ def _estimate_mode_from_histogram(self) -> List[float]:
         mode = (
             bin_edges[highest_idxs] + bin_edges[highest_idxs + 1]  # type: ignore
         ) / 2
-        return mode.tolist()
+        return cast(List[float], mode.tolist())
```

Comment:
is this a list of float or np.float64?

Reply:
`bin_edges` doesn't necessarily need to contain numpy float values.

Reply:
The issue is if we are prescribing it to contain something that isn't true. I'd rather say it could contains either if that is true because otherwise if someone is relying on static typing to ensure functionality of a function, this could be cause them to infer incorrectly.

Reply:
Done

Reply:
I did some more investigating and numpy functions end up converting the values to `float64` so this should only be `List[np.float64]`

Reply:
Actually it would be `List[float]` because `.tolist()` converts values to the compatible python type.